<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Preset</class>
 <widget class="QDialog" name="Dialog_Preset">
  <property name="windowTitle">
   <string>Preset Folders</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>565</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>7</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item row="11" column="0">
       <widget class="QLabel" name="label_50">
        <property name="text">
         <string>Folder 10:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="folder6">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="6">
       <widget class="QToolButton" name="folder9_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QToolButton" name="folder2_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_43">
        <property name="text">
         <string>Folder 3:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_44">
        <property name="text">
         <string>Folder 4:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="6">
       <widget class="QToolButton" name="folder11_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QToolButton" name="folder7_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_49">
        <property name="text">
         <string>Folder 9:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_51">
        <property name="text">
         <string>Folder 11:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="folder3">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLineEdit" name="folder12">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="folder1">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="11" column="6">
       <widget class="QToolButton" name="folder10_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="9" column="6">
       <widget class="QToolButton" name="folder8_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLineEdit" name="folder11">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_54">
        <property name="text">
         <string>Folder 14:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_52">
        <property name="text">
         <string>Folder 12:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QToolButton" name="folder6_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLineEdit" name="folder8">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="folder2">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>Folder 2:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="2">
       <widget class="QLineEdit" name="folder13">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="13" column="6">
       <widget class="QToolButton" name="folder12_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="15" column="6">
       <widget class="QToolButton" name="folder14_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="folder5">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_53">
        <property name="text">
         <string>Folder 13:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="2">
       <widget class="QLineEdit" name="folder14">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="folder4">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_47">
        <property name="text">
         <string>Folder 7:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="6">
       <widget class="QToolButton" name="folder13_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_48">
        <property name="text">
         <string>Folder 8:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="folder7">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QToolButton" name="folder4_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_45">
        <property name="text">
         <string>Folder 5:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLineEdit" name="folder9">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QToolButton" name="folder5_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLineEdit" name="folder10">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_46">
        <property name="text">
         <string>Folder 6:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QToolButton" name="folder3_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_41">
        <property name="text">
         <string>Folder 1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="folder1_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="16" column="2">
       <widget class="QLineEdit" name="folder15">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="16" column="6">
       <widget class="QToolButton" name="folder15_TB">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_55">
        <property name="text">
         <string>Folder 15:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>5</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_101">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_102">
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_PB">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_103">
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>25</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_PB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_104">
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>folder1</tabstop>
  <tabstop>folder1_TB</tabstop>
  <tabstop>folder2</tabstop>
  <tabstop>folder2_TB</tabstop>
  <tabstop>folder3</tabstop>
  <tabstop>folder3_TB</tabstop>
  <tabstop>folder4</tabstop>
  <tabstop>folder4_TB</tabstop>
  <tabstop>folder5</tabstop>
  <tabstop>folder5_TB</tabstop>
  <tabstop>folder6</tabstop>
  <tabstop>folder6_TB</tabstop>
  <tabstop>folder7</tabstop>
  <tabstop>folder7_TB</tabstop>
  <tabstop>folder8</tabstop>
  <tabstop>folder8_TB</tabstop>
  <tabstop>folder9</tabstop>
  <tabstop>folder9_TB</tabstop>
  <tabstop>folder10</tabstop>
  <tabstop>folder10_TB</tabstop>
  <tabstop>folder11</tabstop>
  <tabstop>folder11_TB</tabstop>
  <tabstop>folder12</tabstop>
  <tabstop>folder12_TB</tabstop>
  <tabstop>folder13</tabstop>
  <tabstop>folder13_TB</tabstop>
  <tabstop>folder14</tabstop>
  <tabstop>folder14_TB</tabstop>
  <tabstop>folder15</tabstop>
  <tabstop>folder15_TB</tabstop>
  <tabstop>save_PB</tabstop>
  <tabstop>cancel_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
